includes/theme-support/class-llms-twenty-twenty-one.php

```diff
@@ -0,0 +1,333 @@
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public static function add_inline_editor_styles() {
+		wp_add_inline_style( 'twenty-twenty-one-custom-color-overrides', self::generate_inline_styles( 'editor' ) );
+	}
+
+	/**
+	 * Enqueue inline styles on the frontend
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public static function add_inline_styles() {
+		wp_add_inline_style( 'twenty-twenty-one-style', self::generate_inline_styles() );
+	}
+
+	/**
+	 * Adds the 2021 theme max-width class around the catalog loop element.
+	 *
+	 * This adds catalog theme support.
+	 *
+	 * @since [version]
+	 *
+	 * @param string[] $classes List of css classes.
+	 * @return string[]
+	 */
+	public static function add_max_width_class( $classes ) {
+		$classes[] = 'default-max-width';
+		return $classes;
+	}
+
+	/**
+	 * Add 2021 theme classes to the LLMS pagnation element
+	 *
+	 * Makes the pagination on catalogs look like the 2021 pagination on post type archives
+	 *
+	 * @since [version]
+	 *
+	 * @param string[] $classes List of css classes.
+	 * @return string[]
+	 */
+	public static function add_pagination_classes( $classes ) {
+		$classes[] = 'navigation';
+		$classes[] = 'pagination';
+		return $classes;
+	}
+
+	/**
+	 * Generate inline CSS for a given context
+	 *
+	 * @since [version]
+	 *
+	 * @param string|null $context Inline CSS context. Accepts "editor" to define styles loaded within the block editor or `null` for frontend styles.
+	 * @return string
+	 */
+	protected static function generate_inline_styles( $context = null ) {
+
+		$selector_prefix = ( 'editor' === $context ) ? '.editor-styles-wrapper' : '';
+
+		$styles = array();
+
+		// Frontend only.
+		if ( is_null( $context ) ) {
+
+			// Fix alignment of content in an access plan.
+			$styles[] = '.llms-access-plan-description ul { padding-left: 0; }';
+
+			// Fix checkboxes.
+			$styles[] = '.llms-form-field.type-checkbox input { width: 25px; display: inline-block; }';
+
+			// Donuts.
+			$styles[] = '.llms-donut svg path { stroke: var( --global--color-secondary ); }';
+
+			// Hide header/footer on certificate pages.
+			$styles[] = '.single-llms_certificate .site-header, .single-llms_my_certificate .site-header, .single-llms_certificate .widget-area, .single-llms_my_certificate .widget-area { display: none; }';
+
+			// Question layout.
+			$styles[] = '.llms-question-wrapper ol.llms-question-choices li.llms-choice .llms-choice-text { width: calc( 100% - 110px); }';
+
+			// Payment gateway stylized radio buttons.
+			$styles[] = LLMS_Theme_Support::get_css(
+				array( '.llms-form-field.type-radio input[type=radio]:checked+label:before' ),
+				array(
+					'background-image' => '-webkit-radial-gradient(center,ellipse,var( --global--color-secondary ) 0,var( --global--color-secondary ) 40%,#fafafa 45%)',
+					'background-image' => 'radial-gradient(ellipse at center,var( --global--color-secondary ) 0,var( --global--color-secondary ) 40%,#fafafa 45%)',
```

Comment:
Ok this doesn't work as we want, of course ehhe, we have an array with 2 equal keys, so the second overrides the first, hence no fall-back. I'd say:
```suggestion
					'background-image' => array(
						'-webkit-radial-gradient(center,ellipse,var( --global--color-secondary ) 0,var( --global--color-secondary ) 40%,#fafafa 45%)',
						'radial-gradient(ellipse at center,var( --global--color-secondary ) 0,var( --global--color-secondary ) 40%,#fafafa 45%)',
					),
```
with the change proposed in the `LLMS_Theme_Support::get_css()`.

Reply:
good call, I like it. I'm going to add a test case to cover this though